Formula/k3d.rb

```diff
@@ -22,31 +22,35 @@ class K3d < Formula
   depends_on "go" => :build
 
   def install
+    require "net/http"
+    uri = URI("https://update.k3s.io/v1-release/channels")
+    resp = Net::HTTP.get(uri)
+    resp_json = JSON.parse(resp)
+    k3s_version = resp_json["data"].find { |channel| channel["id"]=="stable" }["latest"].sub("+", "-")
+
     system "go", "build",
            "-mod", "vendor",
-           "-ldflags", "-s -w -X github.com/rancher/k3d/v#{version.major}/version.Version=v#{version}"\
-                       " -X github.com/rancher/k3d/v#{version.major}/version.K3sVersion=latest",
-           "-trimpath", "-o", bin/"k3d"
+           *std_go_args(ldflags: "-s -w -X github.com/rancher/k3d/v#{version.major}/version.Version=v#{version} -X github.com/rancher/k3d/v#{version.major}/version.K3sVersion=#{k3s_version}")
 
     # Install bash completion
-    output = Utils.safe_popen_read("#{bin}/k3d", "completion", "bash")
+    output = Utils.safe_popen_read(bin/name, "completion", "bash")
```

Comment:
Can we just use the more verbose
```suggestion
    output = Utils.safe_popen_read(bin/"k3d", "completion", "bash")
```

Reply:
:heavy_check_mark:

Reply:
I do think `name` is pretty appropriate here so that `brew extract` just works without any tweaks given that we're using `std_go_args`.

Reply:
@carlocab , is there any action needed still or can this be merged? :)